src/test/java/org/calrissian/flowmix/api/aggregator/CountAggregatorTest.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2015 Calrissian.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.calrissian.flowmix.api.aggregator;
+
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+/**
+ * @author Miguel A. Fuentes Buchholtz
```

Comment:
Generally, @author tags are put in so that there's accountability for the creation of a chunk of code. In many OS communities, the accountability is on the community itself and the ownership is shared amongst whomever wants to take the responsibility for fixing bugs as they pop up. It's no biggie, just a little bit of a paradigm shift from the way of private companies.